src/Data/ProgrammesDb/Entity/Programme.php

```diff
@@ -129,4 +136,14 @@ public function setFirstBroadcastDate(?DateTime $firstBroadcastDate): void
     {
         $this->firstBroadcastDate = $firstBroadcastDate;
     }
+
+    public function getIsGap(): bool
```

Comment:
AFAIR (Remember :D) doctrine also tries (isGap()) ? Which I think might be better? No?

Reply:
We use getIs...() everywhere else, Doctrine isn't doing any magic here, it's just a getter

Reply:
Alright got it, then better to make it consistent, sorry